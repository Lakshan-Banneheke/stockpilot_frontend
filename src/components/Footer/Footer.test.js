import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import React from 'react'

import Footer from './Footer'

test('matches snapshot', () => {
  const wrapper = shallow(<Footer />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
