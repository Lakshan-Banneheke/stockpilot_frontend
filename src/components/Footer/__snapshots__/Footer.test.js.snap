// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <Styled(MuiBox)
    bgcolor="#fff"
    borderTop={1}
    color="#020115"
    px={
      Object {
        "sm": 10,
        "xs": 2,
      }
    }
    py={
      Object {
        "sm": 5,
        "xs": 2,
      }
    }
  >
    <WithStyles(ForwardRef(Grid))
      container={true}
      spacing={5}
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
        sm={4}
        xs={12}
      >
        <WithStyles(ForwardRef(Grid))
          component={
            Object {
              "$$typeof": Symbol(react.forward_ref),
              "propTypes": Object {
                "innerRef": [Function],
                "onClick": [Function],
                "replace": [Function],
                "target": [Function],
                "to": [Function],
              },
              "render": [Function],
            }
          }
          to="/"
        >
          <img
            alt="logo"
            className="makeStyles-logo-3"
            src="logo.svg"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Typography))>
          The website for all Your Crypto and Stock Analysis needs
        </WithStyles(ForwardRef(Typography))>
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Grid))
        className="makeStyles-gridItem-1"
        item={true}
        sm={4}
        xs={12}
      >
        <WithStyles(ForwardRef(Typography))>
          Helpful Links
        </WithStyles(ForwardRef(Typography))>
        <div
          className="makeStyles-gridItemLinks-2"
        >
          <Link
            className="makeStyles-link-4"
            to="/"
          >
            Home
          </Link>
          <Link
            className="makeStyles-link-4"
            to="/crypto"
          >
            Crypto
          </Link>
          <Link
            className="makeStyles-link-4"
            to="/stock"
          >
            Stock
          </Link>
        </div>
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Grid))
        className="makeStyles-gridItem-1"
        item={true}
        sm={4}
        xs={12}
      >
        <WithStyles(ForwardRef(Typography))>
          Contact Us
        </WithStyles(ForwardRef(Typography))>
        <div
          style={
            Object {
              "marginTop": 10,
            }
          }
        >
          <Memo(ForwardRef(FacebookIcon))
            className="makeStyles-icon-5"
          />
          <Memo(ForwardRef(TwitterIcon))
            className="makeStyles-icon-5"
          />
          <Memo(ForwardRef(LinkedInIcon))
            className="makeStyles-icon-5"
          />
        </div>
        <WithStyles(ForwardRef(Grid))
          className="makeStyles-mail-7"
          container={true}
        >
          <Memo(ForwardRef(LocationOnIcon))
            style={
              Object {
                "marginRight": 20,
              }
            }
          />
          <WithStyles(ForwardRef(Typography))>
            32/1, Temple Road, Colombo 7
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          container={true}
          style={
            Object {
              "marginTop": 15,
            }
          }
        >
          <Memo(ForwardRef(MailIcon))
            style={
              Object {
                "marginRight": 20,
              }
            }
          />
          <WithStyles(ForwardRef(Typography))>
            info.stockpilot@gmail.com
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
  </Styled(MuiBox)>
</div>
`;
